src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -156,11 +156,6 @@ public bool TryLoadConfig(
                     logger?.LogInformation("Monitoring config: {ConfigFilePath} for hot-reloading.", ConfigFilePath);
                 }
 
-                if (!string.IsNullOrEmpty(defaultDataSourceName))
```

Comment:
if we can remove assigning of defaultDataSourceName, the function definition should remove that parameter since its no longer used. 

And all callers of this function should remove passing this argument along.